libretroshare/src/services/p3gxschannels.cc

```diff
@@ -1036,6 +1042,108 @@ bool p3GxsChannels::getChannelsContent(
 	return getPostData(token, posts, comments);
 }
 
+bool p3GxsChannels::createChannel(RsGxsChannelGroup& channel)
```

Comment:
I don't think one is wrong and the other is right, they offer different behavior, I kept the other with the original name because it have the asynchronous semantic of GXS style API like all other services. The documentation in the header clearly states that that function is blocking, also I would avoid the longer name.